src/components/Projects/Projects.js

```diff
@@ -22,7 +22,7 @@ function Projects({ projects, id, emptyText = 'No projects found.' }) {
               />
               <div className={styles['list-item-content']}>
                 <Heading level="h3">
-                  <Link href={project.uri ?? '#'}>
+                  <Link href={`projects/${project.databaseId}` ?? '#'}>
```

Comment:
Temporarily hardcoding so that links work from `/projects`

Reply:
The query on the Project single page uses `idType='SLUG'`, so perhaps this should be `project.slug`? (The mismatch appears to work since ACM uses the post ID as the slug by default)

Reply:
It may be worthy here to leave an inline comment as to why we are doing it this way just for the projects CPT.

Reply:
Yes thank you!